src/asami/core.cljc

```diff
@@ -1,20 +1,22 @@
 (ns ^{:doc "A storage implementation over in-memory indexing. Includes full query engine."
       :author "Paula Gearon"}
     asami.core
-    (:require [clojure.string :as str]
-              [asami.storage :as storage :refer [ConnectionType DatabaseType]]
-              [asami.memory :as memory]
-              [asami.query :as query]
-              [asami.internal :as internal]
-              [asami.datom :as datom :refer [->Datom]]
-              [asami.graph :as gr]
-              [zuko.util :as util]
-              [zuko.node :as node]
-              [zuko.entity.general :as entity :refer [EntityMap GraphType]]
-              [zuko.entity.writer :as writer :refer [Triple]]
-              [zuko.entity.reader :as reader]
-              #?(:clj  [schema.core :as s]
-                 :cljs [schema.core :as s :include-macros true])))
+  (:require [clojure.string :as str]
```

Comment:
This isn't the default indentation style of this block (following the standard that emacs goes with). I'd prefer not to merge this, since I'll just go back and immediately change it back.

I'm good with the new `:import` section

Reply:
Odd, I am using emacs well. I will fix it once I am in front of the computer again (tomorrow morning).

Reply:
🤷‍♀️
I just noticed a file (a test file in a branch) that had the name indented by 4 spaces, and the `(:require...` block was only indented by 2, just like your code. But when I reformatted, it corrected it to 4 spaces. No idea why that happened.

Reply:
I had do disable aggressive-indent to make it comply with the current indentation. I am not sure why we have differences really. aggressive-indent just consistently indents the code while you modify it so it should not have any impact, I doubt it's this. 

```
GNU Emacs 28.0.50
CIDER 1.0.0 snapshot (cad36751)
clojure-mode 75c2889
```

Reply:
To be fair, I found out what was happening, and why there is inconsistency (I only noticed because you appeared to be changing it unnecessarily).

The standard indent does have `:require` etc indenting by only 2 spaces, with the namespace at 4 spaces. However, when you use meta-Q to correct indent (which I do regularly), then it indents all of it by 4 spaces. This is what I expected anyway. I don't know why emacs is using both.

Thank you for making the change for me.

Reply:
Mystery solved :)

I really like aggressive-indent, I never have to reindent anything.

Thanks for the merge. On a related note I would have rebased the pr into 1 commit but I thought you would do a squash merge (on gh the merge button can be changed to do this by default in case you didn't know).

Reply:
I don't use the GH interface a lot, so I didn't know about that. Thanks.
I tend to let all the commits go through, since it provides a finer grain history of what's happening. You'll see that most of the codebase is just me (so far), so not squashing hasn't been an issue for me before now.